packages/react-model-ui/package.json

```diff
@@ -38,7 +38,7 @@
     "@mui/icons-material": "^5.11.16",
     "@mui/material": "^5.13.5",
     "@mui/x-data-grid": "^6.2.1",
-    "@theia/core": "1.37.2",
+    "@theia/core": "1.43.1",
```

Comment:
This dependency is unexpected I think. The react component shouldn't have theia dependency right?

Reply:
Absolutely correct, I just did a search and replace but very good catch! Interestingly I thought we already got rid of it but good to fix that now, thank you!

Reply:
I thought so to, that's why I found it strange. But that it's gone now :)